docs/en/token_endpoint.rst

```diff
@@ -226,7 +234,7 @@ UserInfo endpoint to get user attributes.
 ID Token
 ++++++++
 
-The ID Token is a JSON Web Token (JWT) that contains information on the user that has executed the authentication. The RPs MUST validate the ID Token.
+The ID Token is a signed JSON Web Token (JWT) that contains information about the user that has executed the authentication. The RPs MUST validate the ID Token.
 
 If the RP exposes in its metadata the parameter **id_token_encrypted_response_alg** the OP MUST encrypt the ID Token.
```

Comment:
questo collide con "until otherwise" modificato dai colleghi agid

@damikael @AntonioFlorio ^

Reply:
Questo è riferito a id_token_encrypted_response_alg?
Per il momento, in SPID il RP non può chiedere la cifratura dell'ID Token, perché non è supportata dagli OP.
Avevamo proposto di mantenere, per il momento, l'indicazione dell'id_token_encrypted_response_alg solo per CIE, per poter inserire la modifica nelle prossime modifiche alle LLGG OIDC.

Reply:
Risolto qui 405291beed015c6148fea39e8daeefbaeb61c366